third_party/CMakeLists.txt

```diff
@@ -54,6 +54,9 @@ else()
     else()
       list(APPEND AWSSDK_LIBRARIES ${aws_cpp_sdk_INSTALL}/lib/lib${lib}.a)
     endif()
+    if(MSVC)
+      target_compile_options(${lib} PRIVATE /W4 /WX)
```

Comment:
Quick clarification, do we mean to make warnings surface better ? or suppress them ?

Reply:
Ah sorry there's a compiler flag that makes warnings get treated as errors. In this case it's a deprecation warning and just trying to get it to ignore, not sure the right incantation